core/src/main/java/tc/oc/pgm/filters/CarryingItemFilter.java

```diff
@@ -1,13 +1,33 @@
 package tc.oc.pgm.filters;
 
+import com.google.common.collect.ImmutableList;
+import java.util.Collection;
+import org.bukkit.event.Event;
+import org.bukkit.event.inventory.InventoryClickEvent;
+import org.bukkit.event.inventory.InventoryDragEvent;
+import org.bukkit.event.player.PlayerDropItemEvent;
+import org.bukkit.event.player.PlayerItemBreakEvent;
+import org.bukkit.event.player.PlayerPickupItemEvent;
 import org.bukkit.inventory.ItemStack;
 import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.kits.ApplyKitEvent;
 
 public class CarryingItemFilter extends ParticipantItemFilter {
   public CarryingItemFilter(ItemStack base) {
     super(base);
   }
 
+  @Override
+  public Collection<Class<? extends Event>> getRelevantEvents() {
+    return ImmutableList.of(
+        PlayerPickupItemEvent.class,
+        PlayerDropItemEvent.class,
+        ApplyKitEvent.class,
+        InventoryClickEvent.class,
+        InventoryDragEvent.class,
+        PlayerItemBreakEvent.class);
+  }
```

Comment:
Isn't there a PGM generifying event called InventoryTransferItemEvent or something along the lines that would trigger under most if not all of these circumstances?